jabgui/src/main/java/org/jabref/gui/walkthrough/declarative/NavigationPredicate.java

```diff
@@ -154,27 +155,28 @@ static <T extends Event> EventHandler<T> decorate(
     }
 
     /// Navigates to the target node by consuming the event.
-    /// 1. Execute beforeNavigate, which typically used to stop node polling and prevent auto-fallback
-    /// navigation during the handler execution.
+    /// 1. Execute beforeNavigate, which typically used to stop node polling and prevent
+    /// auto-fallback navigation during the handler execution.
     /// 2. Execute the original event handler if it exists.
     /// 3. Execute the onNavigate runnable, when
     ///    - The original handler has completed execution, or
     ///    - The handler has timed out after HANDLER_TIMEOUT_MS milliseconds.
-    ///    - A new window has been opened, or an existing window has been closed.
+    ///    - A new window has been opened.
     ///
-    ///    Those conditions ensure that we will still navigate if original handler is blocking (e.g., showing a dialog,
-    ///    and we are highlighting a node in the dialog) and is still responsive (i.e., we will navigate after a certain
-    ///    amount of time, or a new window has been opened, and we can start polling the new window).
+    ///    These conditions ensure that we will still navigate if original handler is
```

Comment:
(Same indent for all lines)

```suggestion
    /// These conditions ensure that we will still navigate if original handler is
```